StardewArchipelago/Items/Mail/ModLetterActions.cs

```diff
@@ -3,22 +3,23 @@
 using System.Collections.Generic;
 using StardewValley;
 using StardewValley.Tools;
+using StardewArchipelago.Stardew;
 
 namespace StardewArchipelago.Items.Mail
 {
     public class ModLetterActions
     {
-        public void AddModLetterActions(Dictionary<string, Action<string>> letterActions)
+        
+        public void AddModLetterActions(Dictionary<string, Action<string>> letterActions, StardewItemManager _stardewItemManager)
```

Comment:
You should hold the stardew item manager in a field of the class. In the current way, you're making lazy evaluated lambdas that rely on a state without that state being explicit.

For example, if this method was made static, everything would blow up and you'd never figure out why